refs/high-level-architecture/readme.md

```diff
@@ -45,138 +52,180 @@ As such it needs to support the following functions:
-These abstractions should follow the [Bridge Pattern](https://en.wikipedia.org/wiki/Bridge_pattern).
+- A System that uses Keycloak for user identity management, CAN use our Data Capture component in combination with 3rd party Encryption library. The System CAN instantiate and use our Privacy Computation Engine, and implement their own Storage able to automatically interpret and enforce actions resulting as output of the DPrivacy Computation Engine (automatic deletion and modification).
 
-This interoperability should support the following (extreme) example scenarios:
+- A System that is essentially a wordpress website, CAN embed the Capture Component in a wordpress webform, use the Storage Component, the Data Consumer Interface (as separate from the wordpress website), use Privacy Computation Engine and embed the Privacy Request Capture Interface in the wordpress website respecting its look and feel.
 
-- A system that uses Keycloak for user identity management, can use our Data Capture component in combination with 3rd party Encryption library. The system can instantiate and use our Data Rights Computation Engine, and implement their own Storage able to automatically interpret and enforce actions resulting as output of the Data Rights Computation Engine (automatic deletion and modification).
-- A system that is essentially a wordpress website, can embed the Capture Component in a wordpress webform, use the Storage Component, the Data Consumer Interface (as separate from the wordpress website), use Data Rights Computation Engine and embed the A Data Rights Request Capture Interface in the wordpress website respecting its look and feel.
-- A system that has their own user identity management, uses 3rd party e2ee library, has its own storage and interfaces for data view, can instantiate the Data Rights Computation Engine and use only the parts of the Data Consumer Interface related to Data Rights Request parametration and approval.
+- A System that has their own user identity management, uses 3rd party e2ee library, has its own storage and interfaces for data view, CAN instantiate the Privacy Computation Engine and use only the parts of the Data Consumer Interface related to Privacy Request parametrization and approval.
 
 ### Separation of Form and Function
 
-The components of the system should be made in such a way that a client system can use only the functional part and implement its own interfaces or otherwise achieve its own desired look and feel and UX.
+The components of [blindnet devkit](../lexicon/RFC-Lexicon-2.md#blindnet-devkit) MUST be made in such a way that a client [Systems](../lexicon/RFC-Lexicon-2.md#system) CAN use only the functional part and implement its own interfaces or otherwise achieve its own desired look and feel and UX.
```

Comment:
```suggestion
The components of [blindnet devkit](../lexicon/RFC-Lexicon-2.md#blindnet-devkit) MUST be made in such a way that client [Systems](../lexicon/RFC-Lexicon-2.md#system) CAN use only the functional part and implement its own interfaces or otherwise achieve its own desired look and feel and UX.
```